Azure/azure-cli
# [ARM] BREAKING CHANGE: Add user confirmation for az ts create

**Description**  

This PR adds a user confirmation flag to az ts create to account for the scenario where a user updates a template spec version via the create command;  in that case the existing template spec version would be overwritten because create is not a patch like operation. 


**Testing Guide**  

{
1. Create template spec version 

ts create -g {rg} -n {template_spec_name} -v 1.0 -l {resource_group_location} -f "{template_file}" -d {display_name} --description {description} --version-description {version_description}

2. Use the create function to  "update"  the template spec in step one. The description should now be null and the template content changed. 

ts create -g {rg} -n {template_spec_name} -v 1.0 -f "{different_template_file}" --yes

}
